psamm/datasource/sbml.py

```diff
@@ -807,6 +826,12 @@ def write_model(self, file, model):
             reaction_tag.set(self._sbml_tag('reversible'), text_type(
                 equation.direction == Direction.Both).lower())
 
+            lower_str, upper_str = self._get_flux_bounds(
+                eq_id, model, flux_limits, equation)
+
+            reaction_tag.set(self._sbml_tag('lowerFluxBound'), lower_str)
+            reaction_tag.set(self._sbml_tag('upperFluxBound'), upper_str)
```

Comment:
You define a parameter here called `LOWER_BOUND`: https://github.com/zhanglab/psamm/pull/123/commits/a167a0455e8f413c8abdcbe761b56cd4af4eb37c#diff-c8d49b0ea75d56cd1cb1d9648030872dL853 What I mean is that, the `lowerFluxBound` should not contain the numerical value of the lower bound directly but should instead refer to the parameter (i.e. `lowerFluxBound="LOWER_BOUND"`). You have to check this with the specification, though, to make sure. Also I'm not sure if it is valid to use the id `LOWER_BOUND` for every reaction. You may need to change the parameter ID to be unique (again, I'm not entirely sure about this so check with the specification).